src/main/kotlin/Car.kt

```diff
@@ -0,0 +1,42 @@
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+
+class Car() {
+
+
+    fun interestOfDriving(numberOfCars: Int): MutableList<Int> {
+        var ListOfInterestToDrive: MutableList<Int> = mutableListOf()
+        var interestToDrive = 0
```

Comment:
By that logic everything could be a string. :wink: @EdenTewelde is right, Boolean is the best type to express that this is a `true`/`false` variable.